### PR TITLE
deposit: MintedDOIValidator fix

### PR DESCRIPTION
**Pull request by bouzlibop**
* FIX Fixes MintedDOIValidator, so that it correctly checks if DOI was
  already minted for the specific upload. (addresses zenodo/zenodo#285)

Signed-off-by: Adrian Pawel Baran <adrian.pawel.baran@cern.ch>

### CODE REVIEW

File: invenio_deposit/validation_utils.py [hunk 1]
```diff
@@ -183,13 +196,13 @@ def __call__(self, form, field):
 
 
 class InvalidDOIPrefix(object):
-    """
-    Validates if DOI
-    """
+
+    """Validates if DOI."""
```
Comment (jirikuncar) [inline invenio_deposit/validation_utils.py]: `s/Validates/Validate/`

File: invenio_deposit/validation_utils.py [hunk 2]
```diff
@@ -228,12 +242,12 @@ def __call__(self, form, field):
 
 
 class MintedDOIValidator(object):
-    """
-    Validates if DOI
-    """
+
+    """Validates if DOI."""
```
Comment (jirikuncar) [inline invenio_deposit/validation_utils.py]: `s/Validates/Validate/`

**Comment by jirikuncar** [comment]
:+1:

**Comment by bouzlibop** [comment]
Thanks @jirikuncar!

**Pull request merged**